torchmetrics/utilities/data.py

```diff
@@ -151,35 +151,6 @@ def get_num_classes(
     return num_classes
 
 
```

Comment:
@SkafteNicki @Borda tagging you both before making the decision to completely remove ``_stable_1d_sort``. Can't see the usage anywhere else.

Reply:
fine by me :]